webapp/pages/code-of-conduct.vue

```diff
@@ -25,7 +25,7 @@
 
       <p>
         {{ $t('code-of-conduct.get-help') }}
-        <a class="hc-editor-link-blot" href="moderation@human-connection.org" target="_blank">
+        <a class="hc-editor-link-blot" href="mailto:moderation@human-connection.org" target="_blank">
```

Comment:
Hey @Ruben80 it looks as if this branch introduces the same changes as #3133. Probably you wanted to commit/push something else?